callback/callback.go

```diff
@@ -40,3 +40,14 @@ func (c *Datastore) Query(q dsq.Query) (dsq.Results, error) {
 	c.F()
 	return c.D.Query(q)
 }
+
+// DiskUsage returns the size if the child datastore, or 0 if it's not
+// a PersistentDatastore.
+func (c *Datastore) DiskUsage() (uint64, error) {
+	c.F()
+	if pd, ok := c.D.(ds.PersistentDatastore); ok {
+		return pd.DiskUsage()
+	}
+
```

Comment:
See comments from previous review.

I would also refactor the logic info a helper function.

(Now GitHub is annoying me, This is the third time I wrote this comment.!)

Reply:
Would you create a separate util module just for this helper?

Reply:
I would just put in in the top-level package and give it would look something like
```
func DiskUsage(d *Datastore) (uint64, error) {
	pd, ok := d.(PersistentDatastore)
	if !ok {
		return 0, errors.New("Unimplemented")
	}
	du, err := pd.DiskUsage()
	if err != nil {
		return 0, err
	}
	log.Printf("%s: DiskUsage: %d\n", d.Name, du)
	return du, nil
}
```
And then in the wrapper methods just use:
```
return DiskUsage(c.D)
```

Reply:
ok

Reply:
(Note this will likely be overkill, if we decide not to return an error.)